aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp

```diff
@@ -2820,12 +2820,23 @@ void svd_magma(const Tensor& A,
                    .mT();
   // U, S, Vh, info are the right size and strides, but are on GPU
   // We copy them into CPU in pinned_memory
-  const auto empty_like_cpu = [](const Tensor& t) {
-    return at::empty_like(t, t.options().device(kCPU).pinned_memory(true));
+  const auto empty_like_cpu = [](const Tensor& t, bool make_contiguous = false) {
```

Comment:
Should this be unguarded without any ROCm-related define? Is this a bug in the implementation?

Reply:
However, this is probably pointing to an issue on the hipSolver side for which we need to file a JIRA.